docs/graph/index.md

```diff
@@ -0,0 +1,48 @@
+---
+title: Windows Community Toolkit - Authentication and Graph
+author: shweaver-MSFT
+description: Authentication providers and Graph powered helpers that make it easy to work with Microsoft Graph APIs.
+keywords: uwp, wpf, netstandard, windows, community, toolkit, graph, login, authentication, provider, providers, identity
+dev_langs:
+  - csharp
+---
+
+# Windows Community Toolkit - Authentication and Graph
+
+The authentication and Graph helpers and controls are a part of the [Windows Community Toolkit](https://aka.ms/wct), focused on enabling quick and easy Windows authentication and [Microsoft Graph](https://developer.microsoft.com/en-us/graph/) powered experiences. These controls and helpers make it easy to get users authenticated and start calling Microsoft Graph APIs!
+
+> Note: This new library replaces the `Microsoft.Toolkit.Uwp.UI.Controls.Graph` package; however, it is not backwards compatible nor does it provide all the same features at this time.
+
```

Comment:
Considering we haven't updated this library in 2+ years, and it only supports Graph v4 and not the latest, and we don't actively maintain it... I'm wondering if we need another call-out of that fact in the index/pages as well?

@Richasy you had the last PR, do you know if this is still being used anymore?

Reply:
Graph controls will be moved into the Archive folder sibling to DataGrid and a porting ticket will be opened alongside the others. Before we can proceed, we need to close the Archive folder PR at #594